vibe_code_apps_13/scripts/linux_timing_baseline.sh

```
@@ -1,26 +1,124 @@
+
 record() { echo "$*" | tee -a "$ART/summary.txt"; }
 
+mark_stopped() {
+  echo "stopped" >"$RESULT_FILE"
+  echo "$1" | tee "$ART/stop-reason.txt" >"$STOP_REASON_FILE"
+  record "STOPPED: $1"
+  exit 1
+}
+
+snapshot_mini_process() {
+  local label="$1" out="$ART/process-${label}.txt"
```

Shellcheck (0.11.0)

[warning] This assignment is used again in this 'local', but won't have taken effect. Use two 'local's.

(SC2318)